.github/workflows/package-cli.yml

```diff
@@ -83,30 +94,43 @@ jobs:
         run: git apply --allow-empty ${{ runner.temp }}/version.patch/version.patch
         timeout-minutes: 1
 
-      # Install fuse for building the CLI (libparsec includes platform_mountpoint)
-      - name: Install fuse
+      - name: Use Cross
         shell: bash
         run: |
-          until sudo apt-get -y install fuse3 libfuse3-dev; do
-            echo "Failed to install APT package, retrying...";
-          done
+          dir="$RUNNER_TEMP/cross-download"
+          mkdir "$dir"
+          echo "$dir" >> $GITHUB_PATH
+          cd "$dir"
+          curl -LO "https://github.com/cross-rs/cross/releases/download/${{ env.cross-version }}/cross-x86_64-unknown-linux-musl.tar.gz"
+          tar xf cross-x86_64-unknown-linux-musl.tar.gz
+          echo "CARGO=cross" >> $GITHUB_ENV
         timeout-minutes: 5
 
       - name: Build CLI
-        run: cargo build -p parsec-cli --release
+        run: >-
+          ${{ env.CARGO }} build -p parsec-cli
+          --release
+          --target=${{ matrix.target }}
+          --features=vendored-openssl
         timeout-minutes: 10
 
+      - name: Debug target folder
```

Comment:
I guess the target folder will contain a lot of stuff (e.g. all the .o files)

so is this really useful to keep, or should we consider this a debugging artifact that can be removed now that the build runs fine ?

Reply:
Now the step is only run if the job enable the debug mode